packages/checkout-extension/src/ExtensionContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { createContext } from 'react';
+
+import { ExtensionService } from './ExtensionService';
+
+export interface ExtensionContextProps {
+    extensionService: ExtensionService;
```

Comment:
Is this something that will keep growing as we add more features?

Reply:
Once it is converted using `useReducer`, we'll only have an extension state.